### PR TITLE
Cirrus Networks should use PoAStandardScriptsRegistry

### PR DESCRIPTION
**Pull request by fassadlr**

### CODE REVIEW

File: src/SwapExtractionTool/Program.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ static async Task Main(string[] args)
             }
             else
             {
-                startBlock = 1_949_800;
+                startBlock = 1975500;
```
Comment (fassadlr) [inline src/SwapExtractionTool/Program.cs:25]: @StratisIain Is it ok to update the start block here to something closer to when we started the swap?

File: src/SwapExtractionTool/SwapExtractionService.cs [hunk 2]
```diff
@@ -108,7 +108,8 @@ private async Task ScanForSwapTransactionsAsync(int startBlock)
                 ProcessBlockForSwapTransactions(block, height);
             }
 
-            Console.WriteLine($"{this.swapTransactions.Count} swap transactions to process...");
+            Console.WriteLine($"{this.swapTransactions.Count} swap transactions to process.");
```
Comment (fassadlr) [inline src/SwapExtractionTool/SwapExtractionService.cs:111]: Sorry, just some small changes to the swap extraction tool thrown in.

File: src/NBitcoin/Policy/StandardTransactionPolicy.cs [hunk 3]
```diff
@@ -48,11 +48,11 @@ public class StandardTransactionPolicy : ITransactionPolicy
         /// </summary>
         public bool CheckScriptPubKey { get; set; }
 
-        private readonly Network network;
+        public readonly Network Network;
```
Comment (quantumagi) [inline src/NBitcoin/Policy/StandardTransactionPolicy.cs:51]: Could this be `protected`?

File: src/SwapExtractionTool/Properties/launchSettings.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "SwapExtractionTool": {
       "commandName": "Project",
-      "commandLineArgs": "-vote"
+      "commandLineArgs": "-swap"
```
Comment (quantumagi) [inline src/SwapExtractionTool/Properties/launchSettings.json:5]: Is this an intentional change?

Comment (fassadlr) [reply src/SwapExtractionTool/Properties/launchSettings.json:5]: Yes :)

File: src/SwapExtractionTool/DistributedSwapTransaction.cs [hunk 5]
```diff
@@ -2,7 +2,7 @@
 {
     public sealed class DistributedSwapTransaction
     {
-        public int BlockHeight { get; set; }
+        public long BlockHeight { get; set; }
```
Comment (quantumagi) [inline src/SwapExtractionTool/DistributedSwapTransaction.cs:5]: We have about 7 times more instances of `int` block heights than `long`. `int` has a maximum value of `2,147,483,647`.

File: src/SwapExtractionTool/SwapTransaction.cs [hunk 6]
```diff
@@ -2,7 +2,7 @@
 {
     public sealed class SwapTransaction
     {
-        public int BlockHeight { get; set; }
+        public long BlockHeight { get; set; }
```
Comment (quantumagi) [inline src/SwapExtractionTool/SwapTransaction.cs:5]: Same here.

**Review by quantumagi — APPROVED** [review]
Looks good. Just a few comments.

File: src/SwapExtractionTool/DistributedSwapTransaction.cs (hunk 5 above)
Comment (fassadlr) [reply src/SwapExtractionTool/DistributedSwapTransaction.cs]: Sorry this change should be reverted.

**Pull request merged**